image/tools/lib/utils.sh

```diff
@@ -5,7 +5,12 @@ function cp_pod_data {
     cp_dest=$2
 
     num_attempted_copy=0
-    max_tries=5
+    max_tries=3
+
+    # Disable errors because files contains may change during backups, which is acceptable and expected in production
+    # JIRA: https://issues.redhat.com/browse/INTLY-10129
+    set +eo pipefail
```

Comment:
This isn't optimal, as it could swallow any other errors that might be more serious. Also the warning message about files changing doesn't result in an error (at least not usually).